src/Aspire.Hosting.AWS/CloudFormation/CloudFormationProvisioner.cs

```diff
@@ -177,4 +182,27 @@ private static IAmazonCloudFormation GetCloudFormationClient(ICloudFormationReso
             throw new AWSProvisioningException("Failed to construct AWS CloudFormation service client to provision AWS resources.", e);
         }
     }
+
+    internal static ImmutableArray<UrlSnapshot>? MapCloudFormationStackUrl(IAmazonCloudFormation client, string stackId)
+    {
+        try
+        {
+            var endpointUrl = client.DetermineServiceOperationEndpoint(new DescribeStacksRequest { StackName = stackId })?.URL;
+            if (endpointUrl == null || !endpointUrl.Contains(".amazonaws."))
+            {
+                return null;
+            }
+
+            var url = $"https://console.aws.amazon.com/cloudformation/home?region={client.Config.RegionEndpoint.SystemName}#/stacks/resources?stackId={stackId}";
+
+            return
```

Comment:
Once we know the region run it through the `^(us|eu|ap|sa|ca|me|af|il)\\-\\w+\\-\\d+$` regex. That will ensure the region is valid for the `console.aws.amazon.com` console link. If the region is not a match then skip adding the URL. The SDK has this regex to figure out what AWS partition the region is in. https://github.com/aws/aws-sdk-net/blob/main/sdk/src/Core/partitions.json#L12C22-L12C63

Reply:
Done. Should there be support for other partitions? I have no familiarity with their URLs so would need some guidance with that.